checkstyle-tester/projects-to-test-imports.properties

```diff
@@ -0,0 +1,12 @@
+# List of GIT repositories to clone / pull for checking with Checkstyle
+# File format: REPO_NAME|[github|git|hg]|REPO_GIT_URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+# Please note that bash comments works in this file
+
+# option=TOP separated=true
+guava|git|https://github.com/google/guava|v25.0|guava/guava-gwt/test/com/google/common/GwtTestSuite.java|
+
+# option=TOP separated=true groups=org,java staticGroups=org,java
+#java-design-patterns|git|https://github.com/iluwatar/java-design-patterns|dd855a376bc025aa61f6816584f79eb9854fe5d7||
```

Comment:
it will be hard to remmember on how to use.

can we make in folder `checkstyle-tester/checks-import-order/`:
```
projects-to-test-imports-guava.properties
checks-imports-error-guava.xml
projects-to-test-imports-java-design-patterns.properties
checks-imports-error-java-design-patterns.xml
projects-to-test-imports-apache-jsecurity.properties
checks-imports-error-apache-jsecurity.xml
launch.sh (where just launch all that config+property sequentially, by diff.groovy, if any diff detected ... exit with non zero code)
```

Reply:
>can we make in folder checkstyle-tester/checks-import-order/

done.